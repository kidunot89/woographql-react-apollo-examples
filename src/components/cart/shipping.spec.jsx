import React from 'react';
import '@testing-library/react/dont-cleanup-after-each';
import { render, cleanup } from '@testing-library/react';

import Shipping from './shipping';
import ApolloMockProvider from '../../testing/apollo-mock-provider';
import SessionProvider from '../session-provider';

describe('Shipping', () => {
  afterAll(() => cleanup());
  const { baseElement } = render(
    <ApolloMockProvider>
      <SessionProvider>
        <Shipping />
      </SessionProvider>
    </ApolloMockProvider>,
  );

  it('should render successfully', () => {
    expect(baseElement).toMatchSnapshot();
  });
});
