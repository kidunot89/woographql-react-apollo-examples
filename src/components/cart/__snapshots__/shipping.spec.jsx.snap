// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shipping should render successfully 1`] = `
<body>
  <div />
</body>
`;
